scripts/release.js

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console */
+/* eslint-disable no-process-exit */
+/* eslint-disable import/no-commonjs */
+
+const { execSync } = require('child_process');
+const colors = require('colors/safe');
+const inquirer = require('inquirer');
+
+// check if user can publish new version to npm
+try {
+  execSync('$(npm owner add `npm whoami`)');
+} catch (e) {
+  console.log(
+    colors.red(`
+    You are not an owner of the npm repository,
+    ask for it before trying to bundle a release.
+  `)
+  );
+  process.exit(1);
+}
+
+// check if branch is clean with all changes commited
+const uncommitedChanges = execSync('git status --porcelain').toString().trim();
+if (uncommitedChanges) {
+  console.log(
+    colors.red(`
+    Working tree is not clean, please commit all your changes before release.
+  `)
+  );
+  process.exit(1);
+}
+
+const currentBranch = execSync('git rev-parse --abbrev-ref HEAD')
+  .toString()
+  .trim();
+
+// check if we are not on branch master or exit
+if (currentBranch === 'master') {
+  console.log(
+    colors.red(`
+    It's not possible to release a new version from the master branch.
+    Please checkout any other branch:
+      - develop: release new stable version
+      - xxxxxxx: release new beta version
+  `)
+  );
+  process.exit(1);
+}
+
+const strategy = currentBranch === 'develop' ? 'stable' : 'beta';
```

Comment:
I don't quite understand this line, does it means when on develop you can only do stable releases and on any other branch (but master), do a beta release?